src/CMakeLists.txt

```diff
@@ -0,0 +1,111 @@
+set(definitions "UNDERSCORE" "NFILES=32" "MAXCD=250" "MAXNC=600" "MXNAF=3")
+add_compile_definitions(definitions)
+
+if(CMAKE_C_COMPILER_ID MATCHES "^(Intel)$")
+  set (FREAL8 "-r8")
+  set (FINT8 "-i8")
+  set (TRACEBACK "-traceback")
+  set (MMMED "-mcmodel=medium")
+  set (IntelComp TRUE)
+elseif(CMAKE_C_COMPILER_ID MATCHES "^(GNU|Clang|AppleClang)$")
+  set (FREAL8 "-fdefault-double-8" "-fdefault-real-8")
+  set (FINT8 "-fdefault-integer-8")
+  set (TRACEBACK "-fbacktrace")
+  set (MMMED "")
+endif()
+
+
+set(fortran_8_flags ${FREAL8} ${FINT8} ${TRACEBACK})
```

Comment:
fortran_8_flags are the same as fortran_8_DA_flags.  They can be combined and reduced to fortran_8_flags

Reply:
The variables are looped over in the kind loop, so need to be duplicated for that reason.

Reply:
Ok. I understand that.

Also, does GNU build all 6 kinds? In the foreach loop, only Intel compiler is checked and then kind is checked.
I don't understand what is going on with these and do we really need these?  What does the GSI use?